recipes/mailio/all/conanfile.py

```diff
@@ -7,6 +7,7 @@
 
 class mailioConan(ConanFile):
     name = "mailio"
+    version = "0.20.0"
```

Comment:
It's not required, in fact, if there would ever be a different version - it'd prevent the recipe from building with it.

Reply:
Yes, @Elnee please remove this line

Reply:
Done